cmd/example-mysql/main.go

```diff
@@ -132,33 +132,35 @@ func main() {
 // "/tile/0/x001/x234/067.p/8" means level 0, index 1234067 and width 8 of a partial tile.
 func parseTileLevelIndexWidth(level, index string) (uint64, uint64, uint64, error) {
 	l, err := strconv.ParseUint(level, 10, 64)
-	if err != nil {
+	if l > 63 || err != nil {
```

Comment:
Maybe I should have had more caffeine this morning, but why 63? A comment or a commented constant would be useful.

Reply:
Added a comment. The maximum level is 63 as specified in the tlog-tiles specification.